server/chat-plugins/calculator.ts

```diff
@@ -97,7 +97,12 @@ function solveRPN(rpn: string[]) {
 			if (!resultStack.length) throw new SyntaxError(`Unknown syntax error`);
 			resultStack.push(-resultStack.pop()!);
 		} else if (!"^%*/+-".includes(token)) {
-			let num = Number(token);
+			let num;
+			if (token.endsWith('h')) {
+			num = Number("0x" + token.slice(0, -1)); //replacing h with 0x
+			} else { 
+			num = Number(token); 
+			}
```

Comment:
```suggestion
			if (token.endsWith('h')) token = `0x${token.slice(0, -1)}`;
			let num = Number(token);
```
This is how I would do it (converting `const token` to `let token`).

Reply:
I think this is more readable than `let num;` but if you'd rather do that, please at least indent the inner part of your `if` statements.